JitsiConference.js

```diff
@@ -1470,6 +1470,23 @@ JitsiConference.prototype.grantOwner = function(id) {
     this.room.setAffiliation(participant.getJid(), 'owner');
 };
 
+/**
+ * Revoke owner rights to the participant or local Participant as
+ * the user might want to refuse to be a moderator.
+ * @param {string} id id of the participant to revoke owner rights to.
+ */
+JitsiConference.prototype.revokeOwner = function(id) {
+    const participant = this.getParticipantById(id);
+    const isMyself = this.myUserId() === id;
+
+    if (isMyself) {
+        this.room.setAffiliation(this.room.myroomjid, 'member');
+    } else if (participant) {
+        this.room.setAffiliation(participant.getJid(), 'member');
```

Comment:
We use `member` only when lobby is enabled in the rest of the cases it is `none`.

Reply:
Oh, I see. I have pushed a commit to deal with that.
Thanks! 😄